Auth0/CredentialsManager.swift

```diff
@@ -58,6 +58,26 @@ public struct CredentialsManager {
         return self.storage.setData(NSKeyedArchiver.archivedData(withRootObject: credentials), forKey: storeKey)
     }
 
+    /// Clear credentials stored in keychain
+    ///
+    /// - Returns: Bool outcome of removal success
+    public func clearCredentials() -> Bool {
+        return self.storage.deleteEntry(forKey: storeKey)
+    }
+
+    /// Checks if valid credentials are available
+    ///
+    /// - Returns: Bool outcome of result
```

Comment:
no need to state the Bool type, just say if there are valid and non-expired credentials stored`